code/_global_vars/mapping.dm

```diff
@@ -0,0 +1,2 @@
+// AREA LIST //
+GLOBAL_LIST_EMPTY(map_areas)
```

Comment:
I plan to list cache more objects here, but that will be an even bigger PR so its being waited for